headers.js

```diff
@@ -7,6 +7,47 @@ var ZERO_OFFSET = '0'.charCodeAt(0)
 var USTAR = 'ustar\x0000'
 var MASK = parseInt('7777', 8)
 
+// Divide this number to get high 32 bit (JS unsafe)
+var DIVIDEND_HIGH_32BIT = Math.pow(2, 32)
```

Comment:
Maybe use upcoming BigInt support? Babel 7 already support it...

Reply:
We aren't using babel so this is fine I think :)

Reply:
I'll keep as it is

Reply:
I'll keep as it is